### PR TITLE
Trello 980 acceptable absence page

### PR DESCRIPTION
**Pull request by dnyanesh-sonwane**
image

### CODE REVIEW

File: src/server/record-outcome/views/failed-to-attend.njk [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends "pages/wizard-form.njk" %}
+
+{% set title -%}
+    Was {{ offenderFirstName }}'s absence acceptable?
```
Comment (fofr) [inline src/server/record-outcome/views/failed-to-attend.njk:4]:
```suggestion
    Was {{ offenderFirstName }}’s absence acceptable?
```

File: src/server/record-outcome/views/failed-to-attend.njk [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        items: [
+            {
+                id: 'acceptableAbsence-acceptable',
+                text: 'Yes',
+                value: 'true',
+                checked: acceptableAbsence === true
+            },
+            {
+                id: 'acceptableAbsence-unacceptable',
+                text: 'No, ' +offenderFirstName+ '\'s absence was unacceptable',
```
Comment (fofr) [inline src/server/record-outcome/views/failed-to-attend.njk:31]:
```suggestion
                text: 'No, ' +offenderFirstName+ '’s absence was unacceptable',
```

File: src/server/record-outcome/views/failed-to-attend.njk (hunk 1 above)
Comment (dnyanesh-sonwane) [reply src/server/record-outcome/views/failed-to-attend.njk]: Fixed

File: src/server/record-outcome/views/failed-to-attend.njk (hunk 2 above)
Comment (dnyanesh-sonwane) [reply src/server/record-outcome/views/failed-to-attend.njk]: Fixed

**Comment by dnyanesh-sonwane** [comment]
[Acceptable absence page](https://trello.com/c/hjLQcZYN/980-acceptable-absence-page)

**Review by electrified — APPROVED** [review]

**Pull request merged**